sdk/ai/Azure.AI.Projects/README.md

```diff
@@ -20,6 +20,9 @@ Use the AI Projects client library to:
       - [Retrieve messages](#retrieve-messages)
     - [File search](#file-search)
     - [Function call](#function-call)
+    - [Azure function call](#azure-function-call)
```

Comment:
Do we have enterprise file search in C# also?

Reply:
In C# framework we do not have the vector store samples yet I have created the task to track this issue: https://github.com/orgs/ai-platform-ml-platform/projects/7/views/2?sliceBy%5Bvalue%5D=nirovins_microsoft&pane=issue&itemId=90891840